.gitignore

```diff
@@ -0,0 +1,53 @@
+./old/palettes.lisp
+.gateway.*
+
+# lisp
+*.FASL
+*.fasl
+*.lisp-temp
+*.lx64fsl
+
+.__*
+
+# emacs
+*~
+\#*\#
+/.emacs.desktop
+/.emacs.desktop.lock
+*.elc
+auto-save-list
+tramp
+.\#*
+
+# Org-mode
+.org-id-locations
+*_archive
+
+# flymake-mode
+*_flymake.*
+
+# eshell files
+/eshell/history
+/eshell/lastdir
+
+# elpa packages
+/elpa/
+
+# reftex files
+*.rel
+
+# AUCTeX auto folder
+/auto/
+
+# cask packages
+.cask/
+dist/
+
+# Flycheck
+flycheck_*.el
+
+# server auth directory
+/server/
+
+# projectiles files
+.projectile
```

Comment:
There's a lot of extraneous entries in this `.gitignore` file, don't you think?  I could understand some patterns for fasl files and macOS and emacs litter, but it seems to me that there's too much junk in here.